### ISSUE TITLE
store ion library as a text file (not in code)

### ISSUE THREAD

**Issue by swharden** [swharden/JLJP#10]
Let's modify the Ion class to look up charges and mobilities from a text file rather than storing these values in code. 

Advantages:
* users can easily add or modify ions
* it becomes easier for people to contribute by sending-in new ions
* the Java program and C# program can share a common ion table
* users can add ions by downloading the latest table (and not recompiling)

Tips:
* mobility table format should be easy to modify in a text editor
* ensure the format is easy to parse and load with Java and C#
* The table I made is easy to reformat: https://github.com/swharden/JLJP/issues/3#issuecomment-584306489

**Comment by swharden** [comment]
[LJPcalc](https://github.com/swharden/LJPcalc) is doing this well. The C# code is easy to copy/paste to JAVA if this feature is desired in the future.

The ion table is easy to edit (and store references in):
https://github.com/swharden/LJPcalc/blob/master/src/IonTable.csv

**Comment by dbrogioli** [comment]
It would be excellent to have a database with ion mobilities, which can be improved by users. It should store the following data: ion short name, extended formula, ion mobility, temperature, source. This is also connected to the question about temperature (#2).

**Comment by swharden** [comment]
> It would be excellent to have a database with ion mobilities, which can be improved by users. 

I initially managed this with a CSV file, but it was awkward to edit. Excel kept wanting to convert it to XLS and editing it in a plain text editor was a bit awkward. Further, adding notes (like references, URLs, etc.) easily broke the CSV formatting.

I just refactored LJPcalc to use _markdown_ rather than csv. It's working very well! It's easy to edit in a text editor, it has the ability to add formatting and links, and it looks great when rendered on GitHub:

* [IonTable.md](https://github.com/swharden/LJPcalc/blob/master/src/IonTable.md)
* [IonTable.md](https://raw.githubusercontent.com/swharden/LJPcalc/master/src/IonTable.md) (raw)

The parsing function I wrote in C# should be easily to implement in Java: https://github.com/swharden/LJPcalc/blob/c0bfdc6f26dcf514c9d1a0da8c16bb4681e1cfe3/src/LJPmath/IonTable.cs#L150-L182

> It should store ... temperature. This is also connected to the question about temperature (#2).

I imagine the overwhelming majority of people wish to calculate LJP at ambient temperature. The presence of 3 mobilities for every ion would probably be distracting and it would be easy to accidentally select the wrong one.

One way we could solve this is to add an option to let the user select which ion table file to use. We could provide 3 files containing mobilities at 0, 18, and 25C. This seems reasonable since an LJP calculation
